### ISSUE TITLE
Error in running 

### ISSUE THREAD

**Issue by shahabty** [yifjiang/relative-depth-using-pytorch#2]
Hello,
It seems that the content of links have been changed. In models, there is no .pt files. There are only .t7 files.

I cannot run the code!

Thanks

**Comment by logic1988** [comment]
I encountered the same problem.

**Comment by renschler** [comment]
@logic1988 can you share your model weights?

**Comment by logic1988** [comment]
[Best_model_period1.pt.zip](https://github.com/yifjiang/relative-depth-using-pytorch/files/1551674/Best_model_period1.pt.zip)

**Comment by renschler** [comment]
thanks @logic1988 ! I got the script to run with python 2.7 without cuda, but results weren't too pretty :). I will upload my model weights if I end up training a new one.

**Comment by logic1988** [comment]
@renschler model was saved by cuda mode and must be converted to cpu mode without cuda.

**Comment by renschler** [comment]
I'm pretty sure I did that but could be wrong.
I removed all calls to cuda() throughout the modules and I also used torch.load("Best_model_period1.pt", map_location=lambda storage, loc: storage) which is what was done here: https://github.com/fastai/fastai/commit/1608998781737982060c0bd4507c063983f6fed2#diff-5e8b1ea365dfa465c52567edb0a02a00

**Comment by guvcolie** [comment]
@renschler @logic1988  Hi! Did you train this model from scratch?
I'm training from scratch, but the loss decreases very slowly...  What about yours?

**Comment by yifjiang** [comment]
Hi guys, I have updated this repository and there is a pt file Best_model_period1.pt in /src/results/, which is a pre-trained “Ours” model as is described in the paper.
